### PR TITLE
Added APE14 support in SunPy visualization

### PR DESCRIPTION
**Pull request by yashrsharma44**
This PR adds support for APE14 in SunPy for `visualization/animator/image.py`.

### CODE REVIEW

File: sunpy/visualization/animator/image.py [hunk 1]
```diff
@@ -159,12 +160,12 @@ class ImageAnimatorWCS(ImageAnimator):
     """
     def __init__(self, data, wcs=None, image_axes=[-1, -2], unit_x_axis=None, unit_y_axis=None,
                  axis_ranges=None, **kwargs):
-        if not isinstance(wcs, astropy.wcs.WCS):
+        if not isinstance(wcs, (astropy.wcs.WCS, SlicedLowLevelWCS)):
```
Comment (Cadair) [inline sunpy/visualization/animator/image.py]: This should just be checking for isinstance of the low level base class.

Comment (yashrsharma44) [reply sunpy/visualization/animator/image.py]: Sure!

**Comment by Cadair** [comment]
@yashrsharma44 is this read to go as is, or have you not yet worked through it all with the bugs in the wcsaxes branch etc?

**Comment by yashrsharma44** [comment]
A slight change needs to be done, replacing `pixel_n_dim` with `world_n_dim`, otherwise it's good to go.

**Comment by Cadair** [comment]
Let me know when you think it's ready.

**Comment by yashrsharma44** [comment]
Hey @Cadair, I have tested the plotting with the above modifications in SunPy, and it's working. Since after slicing, we want to reduce the axes that we want to plot, so `pixel_n_dim` serves the purpose, so no change required.
This PR can be merged!

**Pull request merged**